docs/known_issues.md

```diff
@@ -14,10 +14,23 @@ As a workaround, use `tag.turbo_frame` instead of `turbo_frame_tag`.
 
 Note: For the same functionality as `turbo_frame_tag(my_model)`, use `tag.turbo_frame(id: dom_id(my_model))`.
 
-## form_for compatibility
+## `form_for` compatibility
 
 ViewComponent [isn't compatible](https://github.com/viewcomponent/view_component/issues/241) with `form_for` helpers by default.
 
+This means that it isn't supported to pass a form object to a view component.
+It won't render as expected. Some content will be rendered twice.
+
+Consider following options:
+
+- Render a form end-to-end within one view component
+- Render a [classical view partial](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials) within the view component, which includes the form
+- Use [FormBuilder](https://guides.rubyonrails.org/form_helpers.html#customizing-form-builders) to create reusable form components.
+  Note: There are multiple approaches how to implement a FormBuilder:
+  - Classic approach (without ViewComponent)
+  - Using a lightweight implementation of ViewComponent (for example, [`ActsAsComponent`](https://github.com/primer/view_components/blob/main/lib/primer/forms/acts_as_component.rb) at [Primer ViewComponents](https://github.com/primer/view_components))
```

Comment:
I'm not sure `ActsAsComponent` is something we want to mention in the official VC docs. I think of it as a private class that's only designed to be used internally. We could expose it as its own gem, but it lacks many standard VC features like slots.

Reply:
It is just an example. One could do something else. I for one like the last option for example...

We can revise this section, when there is a better solution. But think about someone from the developer/user side. He/she is looking for a solution. So it makes sense to mention all known options

Reply:
I tried to re-word it. Please see, if you like it.

Reply:
Ok I'll take another look. I understand it's an example, I just don't want to support someone using it outside the context it was designed for 😅